lib/metadataTypes/Discovery.js

```diff
@@ -33,7 +33,7 @@ class Discovery extends MetadataType {
                 v.key = Object.keys(this.definition.endPointMapping)[i];
                 metadataStructure[v.key] = v;
             }
-            await this.saveResults(metadataStructure, retrieveDir, null);
+            await super.saveResults(metadataStructure, retrieveDir, null);
```

Comment:
why did you move to super?

Reply:
as discussed, just to signal more clearly that this is actually using the super-version and not a local one. like we've done it in other type-classes.